pymole.py

```diff
@@ -12,7 +12,7 @@
 
 def respond_json(obj):
 	if isgenerator(obj):
-		return Response((json.dumps(o) for o in obj), mimetype='application/stream+json')
+		return Response(obj, mimetype='text/event-stream')
```

Comment:
What about a simple and more generic
```python
return Response(('data: {0}\n\n'.format(json.dumps(o)) for o in obj), mimetype='text/event-stream')```
(or extracting this into a function if you prefer)
A quick test with an JS EventSource seems to work fine. What I like about it is that it can work with any generator, not just with our custom 'Observable'. And we don't mix the JSON encoding stuff into our Observable.